utils/interfaces/_library/miscs.py

```diff
@@ -109,9 +114,40 @@ def check(self, data):
             self.log_error(f"{data['log_filename']} content should be an array")
             return
 
+        check_pass = False
+        span_types = []
```

Comment:
Use a `set()`, it's quite the same as a list, but removes duplicates.

Reply:
We want duplicates in my opinion, it is valuable information to know if there are multiple spans of a type.

Reply:
agree